src/emmy/structure.cljc

```diff
@@ -19,7 +19,9 @@
                             AFn IFn
                             IPersistentVector IReduce IKVReduce
                             IObj
-                            Indexed Reversible Sequential))))
+                            Indexed Reversible Sequential)))
+  #?(:cljs
```

Comment:
Do double require statements work? The pattern I like is sticking a line in the big require block above like

```
#?(:cljs [goog.object :as gobject])
```

Reply:
obsolete

Reply:
✅